libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/feedback/FeedbackBottomSheet.java

```diff
@@ -33,13 +33,25 @@
   Animator.AnimatorListener {
 
   public static final String TAG = FeedbackBottomSheet.class.getSimpleName();
+  public static final long CLOSE_BOTTOM_SHEET_AFTER = 100L;
 
   private FeedbackBottomSheetListener feedbackBottomSheetListener;
   private FeedbackAdapter feedbackAdapter;
   private RecyclerView feedbackItems;
   private ProgressBar feedbackProgressBar;
   private ObjectAnimator countdownAnimation;
   private long duration;
+  private CountDownTimer countDownTimer = new CountDownTimer(CLOSE_BOTTOM_SHEET_AFTER, 0) {
```

Comment:
could this be achieved slightly more elegantly with `hanlder.postDelayed()`?

Reply:
The only reason of using a CountdownTimer is that it is self destroying. My use case is very simple as in you need to dismiss the bottomSheet after some delay. The only reason Handler was not used, is because then I will have to take care of destroying it.
BTW, I am still changing the way I am using the CountDownTimer so as to handle the use case if the user happens to click on multiple feedback immediately.

Reply:
you can register timer on the main looper if `dismiss()` is related to the ui and doesn't have to be run in background thread. then it's a one shot thing.

Reply:
The timer is already running on the main thread. The good thing about CountDownTimer is that it gives a callback(`onFinish`) when it is self destroyed. With Handler I have to make sure that I add calls to removeCallback at appropriate place. It's also very easy to handle the use case where the user click's on two feedback one after another immediately. In that case I can just cancel the timer and start a new one

Reply:
i'm not following why can't we simply do:
```
Handler handler = new Handler(Looper.getMainLooper());
handler.postDelayed(new Runnable() {
   @Override
    public void run() {
        dismiss();
    }
}, 1000L);
```
am i missing something?

Reply:
where do we remove the callback on `handler`
```
Handler handler = new Handler(Looper.getMainLooper());
handler.postDelayed(new Runnable() {
   @Override
    public void run() {
        this.removeCallbacks();
        dismiss();
    }
}, 1000L);
```
How do we handle the use case when the user clicks on 2 feedback immediately. In that case multiple postDelayed calls will spin up.